dev/docgen/.docgenSourceMap.json

```diff
@@ -0,0 +1,46 @@
+[
+    {
+        "source": "src/__snippets__/__tests__/snippets.test.ts",
+        "generated": "/home/shawn/arktype/dev/arktype.io/docs/demos/generated/snippets.test.ts"
```

Comment:
Any time you see local absolute paths in a diff, that's usually a red flag. See the comment below about the expected format.